new-games-to-db/deploy.sh

```
@@ -1,6 +1,39 @@
-#!/bin/bash 
-cd $(realpath "$(dirname "${BASH_SOURCE[0]}")") &&
-    gcloud --project pennantchase-256 \
-	   functions deploy --gen2 --region=us-west1 --runtime=python312 \
-	   --no-allow-unauthenticated \
-	   new_games_to_db --trigger-http
+#!/usr/bin/bash 
+
+set -o pipefail
+
+shopt -s expand_aliases
+source $HOME/.bash_script_aliases
```

Shellcheck (0.10.0)

[info] Not following: ./.bash_script_aliases was not specified as input (see shellcheck -x).

(SC1091)